app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -34,7 +34,7 @@ def button
   def scan_images
     assert_privileges("image_scan")
     showlist = @lastaction == "show_list"
-    ids = showlist ? find_checked_items : find_current_item(ContainerImage)
```

Comment:
I don't see an RBAC check here for the `find_current_item` branch yet I can see it on line 53. Is that correct?